app/actors/hyrax/actors/create_with_files_actor.rb

```diff
@@ -0,0 +1,62 @@
+module Hyrax
+  module Actors
+    # Creates a work and attaches files to the work
+    class CreateWithFilesActor < Hyrax::Actors::AbstractActor
```

Comment:
I think we need to override it to extract file uses attributes from the object metadata and extend method attach_files(files, env, file_uses) to accept the file uses parameter for ingest.